lang/fr-fr.js

```diff
@@ -26,21 +26,21 @@ export default {
 	"components.form-element.defaultError": "{label} n'est pas valide.",
 	"components.form-element.defaultFieldLabel": "Champ",
 	"components.form-element.input.email.typeMismatch": "L'adresse e-mail n'est pas valide.",
-	"components.form-element.input.number.rangeError": "Le nombre doit être {minExclusive, select, true {supérieur à} false {supérieur ou égal à}} {min} et {maxExclusive, select, true {inférieur à} false {inférieur ou égal à}} {max}.",
-	"components.form-element.input.number.rangeOverflow": "Le nombre doit être {maxExclusive, select, true {inférieur à} false {inférieur ou égal à}} {max}.",
-	"components.form-element.input.number.rangeUnderflow": "Le nombre doit être {minExclusive, select, true {supérieur à} false {supérieur ou égal à}} {min}.",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {Le nombre doit être inférieur à {max}.} other {Le nombre doit être inférieur ou égal à {max}.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {Le nombre doit être supérieur à {min}.} other {Le nombre doit être supérieur ou égal à {min}.}}",
 	"components.form-element.input.text.tooShort": "{label} doit contenir au moins {minlength} caractères.",
 	"components.form-element.input.url.typeMismatch": "URL non valide",
 	"components.form-element.valueMissing": "{label} est requis.",
-	"components.form-error-summary.errorSummary": "Il y avait {count, plural, one { une erreur } other { { count } erreurs }} dans les informations que vous avez soumises.",
+	"components.form-error-summary.errorSummary": "{count, plural, one {Il y avait une erreur dans les informations que vous avez soumises.} other {Il y avait {count} erreurs dans les informations que vous avez soumises.}}",
 	"components.input-date-range.endDate": "Date de fin",
 	"components.input-date-range.errorBadInput": "{startLabel} doit être antérieur à {endLabel}",
 	"components.input-date-range.startDate": "Date de début",
 	"components.input-date-time-range-to.to": "à",
 	"components.input-date-time-range.endDate": "Date de fin",
 	"components.input-date-time-range.errorBadInput": "{startLabel} doit être antérieur à {endLabel}",
 	"components.input-date-time-range.startDate": "Date de début",
-	"components.input-date-time.date": "Date",
+	"components.input-date-time.date": "Date", // mfv-translated
```

Comment:
This is a new feature that @bearfriend added where we can indicate that it's actually translated even though the English and in this case French words are the same.